src/ExperimentRandomWeighedVariable.js

```diff
@@ -0,0 +1,22 @@
+import { randomFloat, totalWeight, getWeighedIndex } from './utils';
+const ExperimentRandomWeighedVariable = ({ weights, variants, logger }) => {
```

Comment:
Do you have to pass in variants in this case? We're going to find the group, not the variant to render, right?